akka-actor/src/main/mima-filters/2.5.24.backwards.excludes

```diff
@@ -0,0 +1,2 @@
+# #27411 changes not caught by previous mima versions
+ProblemFilters.exclude[IncompatibleSignatureProblem]("*")
```

Comment:
Really everything?

Reply:
`IncompatibleSignatureProblem` was introduced in this MiMa version, so without this filter it reports on all incompatible signatures between current and the beginning of time.

With this filter in `2.5.24.backwards.excludes` we still have to explicitly add warnings for things that are still different between master and 2.5.25, but not compared to any previous versions. I'm not sure how valuable it is to go back further.

Reply:
Great reasoning!